catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/components/clickup.json

```diff
@@ -0,0 +1,41 @@
+{
+  "component": {
+    "kind": "component",
+    "name": "clickup",
+    "title": "ClickUp",
+    "description": "Receives events from ClickUp webhooks.",
+    "deprecated": false,
+    "firstVersion": "1.0.0-SNAPSHOT",
```

Comment:
This should be 4.9.0-SNAPSHOT

Reply:
It's strange, since that code resulted from the `mvn clean install -DskipTests` command execution.
I'll try to re-execute it once more and see if it gets generated the right way.

Reply:
@oscerd it's not generating the proper version number... maybe it's due to some automatic mechanism for calculating the first version.. could it be that it uses `1.0.0-SNAPSHOT` because the `4.9.0-SNAPSHOT` it's not yet released? Should I edit all the occurrences the version manually? Thanks

Reply:
Set first version in `@UriEndpoint`

Reply:
@davsclaus it's already there... 

https://github.com/nickshoe/camel/blob/472cc0ecbbefff39e0cfb7b96781dbd079102a97/components/camel-clickup/src/main/java/org/apache/camel/component/clickup/ClickUpEndpoint.java#L41